konfera/urls.py

```diff
@@ -19,4 +19,5 @@
 urlpatterns = [
     url(r'^events/', include('konfera.event.urls'), name='event_list'),
     url(r'^register/', include('konfera.register.urls')),
+    url(r'^meetups/', 'views.meetup_list', name='list_all_meetups'),
```

Comment:
this is referencing to non existing view. Please create views.py in the konfera directory and move the view logic from register.views function `meetup_list` into newly created views.